execution_workflows/DaPars2/Dapars2_snakemake/workflow/scripts/results_to_bed.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+
+
+import pandas as pd
+import numpy as np
+import argparse
+import os
+
+
+def proximal_site_to_bed_coords(df,
+                                coord_col="Predicted_Proximal_APA",
+                                one_based=True,
+                                start_outcol="start",
+                                end_outcol="end"):
+    '''
+    Take the predicted proximal poly(A) site in column 'coord_col' (default: "Predicted_Proximal_APA")
+    and return df with columns containing BED compliant start (named start_outcol) & end (named end_outcol) coordinates
```

Comment:
Could @mfansler @mrgazzara have a look whether this is compatible with the summary wf inputs, because there was a discussion earlier about proper bed coordinates for PAS (treat like single nucleotide or treat like indel?) and I'm not sure on what we agreed.